src/tangram.js

```diff
@@ -70,31 +81,32 @@ TC.prototype = {
     }, 0);
   },
 
-  addLayer: function (layer, i) {
-    let config = CCSS.carto2Draw(layer.meta.cartocss, i);
+  addLayer: function (layer) {
+    let layers = CartoCSSRenderer.render(layer.meta.cartocss).getLayers();
 
-    config.forEach(l => {
+    layers.forEach((l, i) => {
+      const yaml = CCSS.carto2Draw(l, i);
       let ly = {
         data: {
           layer: layer.id,
           source: 'CartoDB'
         },
-        draw: l.draw,
-        visible: layer.visible
+        draw: yaml.draw,
+        visible: l.visible
```

Comment:
Should we use `l` or `layer` here?

Reply:
`l` because `layer` contains all the layers... the naming is very confusing, we should refactor this, now or after merging, what do you think?

Reply:
I would clarify this before merging it.

Reply:
Now, I'm not so sure... `layer` contains layers, but at the same time it's possible that there are multiple layers at the `layer` level, not only at the `l` level... That would explain the old line, and then your changes will be correct, do you know how can we test the `visible` property?

Reply:
I checked using Chrome's dev tools how was the input and `layer` is the correct one.